share/cmake/modules/Findyaml-cpp.cmake

```diff
@@ -43,14 +43,21 @@ endif()
 
 if(NOT OCIO_INSTALL_EXT_PACKAGES STREQUAL ALL)
     set(_yaml-cpp_REQUIRED_VARS yaml-cpp_LIBRARY)
-
     if(NOT DEFINED yaml-cpp_ROOT)
         # Search for yaml-cpp-config.cmake
         find_package(yaml-cpp ${yaml-cpp_FIND_VERSION} CONFIG QUIET)
     endif()
 
     if(yaml-cpp_FOUND)
-        get_target_property(yaml-cpp_LIBRARY yaml-cpp LOCATION)
+        if(TARGET yaml-cpp::yaml-cpp)
```

Comment:
Assuming you needed to add `yaml-cpp_INCLUDE_DIR` because the find_package with yaml-cpp 0.8.0 creates a yaml-cpp:yaml-cpp target but the target creation conditional on line 147 only checks for the existence of the old yaml-cpp one so will still re-create a target even if not really needed. Probably still ok if that works correctly on your tests.

Think there is a minor indentation issue for the if / else / endif here.